### PR TITLE
Add recaptcha component

### PR DESCRIPTION
**Pull request by stephankaag**
Currently, there is no support yet for serverside verification.

### CODE REVIEW

**Comment by stephankaag** [comment]
Any feedback? :-)

File: src/components/recaptcha.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            'https://www.google.com/recaptcha/api.js?render=explicit';
+          script.async = true;
+          script.defer = true;
+
+          document.body.appendChild(script);
+
+          return () => {
+            document.body.removeChild(script);
+          };
+        }, []);
```
Comment (vdberganton) [inline src/components/recaptcha.js:59]: This is a very hackie way of using a script for this component and not something that we want to support.

Comment (stephankaag) [reply src/components/recaptcha.js:59]: Do you have a recommended alternative way to include a script tag in a component?

Comment (vdberganton) [reply src/components/recaptcha.js:59]: No I believe this is currently the only solution for using a script in a component. 

It is also possible to add this to a bundle but we don't want to add this as another dependency to our default bundle. 

We will have to figure out a proper solution to be able to support this.

**Comment by vdberganton** [comment]
Closing this now while working towards a solution for adding dependencies

**Pull request closed without merging**